docs/changes.rst

```diff
@@ -21,6 +21,30 @@ Hypothesis APIs come in three flavours:
 You should generally assume that an API is internal unless you have specific
 information to the contrary.
 
+
+------------------
+3.6.1 - 2016-12-20
+------------------
+
+This release fixes a dependency problem and makes some small behind the scenes
+imporvements.
+
+* The fake-factory dependency was renamed to faker. If you were depending on
```

Comment:
Re-reading: I think there’s the potential for confusion here. Should I switch to using `pip install hypothesis[faker]`? Or stick with `pip install hypothesis[fake-factory]`? Or will either work?

Long-term I think we should switch to `hypothesis[faker]` for consistency, not necessarily now – but we should be clear whether that changes right now.